tools/ci-build.sh

```diff
@@ -252,8 +252,8 @@ cmake ${CMAKE_FLAGS} ..
 
 # If CMAKE_ONLY is active, only run CMake. Do not build.
 if [ "$CMAKE_ONLY" == "OFF" ]; then
-  make
-  sudo make install
+  make -j$((`nproc`+1))
```

Comment:
Why don't use ninja (via `-G Ninja`)? And you can just do `cmake --build <dir>` so it could invoke whatever tool was used to generate build files. Ninja scales much better and multi-threaded by default.

Reply:
I think I could say that I never used `make` with p4c. Over the last ~5 years or so :) But YMMV

Reply:
You can do same for `cmake --install` as well, btw.

Reply:
Yes, absolutely! Yeah, I might not have all backends enabled, so this might be  an issue.

Reply:
#4471